openpype/hosts/maya/plugins/publish/extract_gltf.py

```diff
@@ -45,6 +45,10 @@ def process(self, instance):
             "glb": True,
             "vno": True    # visibleNodeOnly
         }
+
+        # convert to gltf shader
+        self.convert_gltf_shader(instance)
+
```

Comment:
This introduces the problem brought up [here](https://github.com/ynput/OpenPype/pull/4261#discussion_r1087555731) again.

Please do **not** merge this PR as is. Publishing should **NOT** automatically change the contents of your current scene. The workfile should remain untouched unless a user triggers an explicit action, like e.g. a Repair Action.

If converted shaders like these are required a Validator should be generated with a Repair action to allow the user to convert it.